internal/lib/lib.go

```diff
@@ -418,29 +418,21 @@ func GetSubnetPrefixInt() int32 {
 	return int32(intCidr)
 }
 
-func GetNetworkNamesForVsVipNode() ([]string, error) {
-	if networkName := GetNetworkName(); networkName != "" {
-		return []string{networkName}, nil
-	} else if IsPublicCloud() && GetCloudType() == CLOUD_AWS {
-		vipNetworkList, err := GetVipNetworkList()
-		if err != nil {
-			return nil, err
-		}
-		if len(vipNetworkList) != 0 {
-			return vipNetworkList, nil
-		}
+/*func GetNetworkNamesForVsVipNode() ([]string, error) {
+	vipNetworkList, err := GetVipNetworkList()
+	if err != nil {
+		return nil, err
 	}
-	return []string{}, nil
-}
-
-func GetNetworkName() string {
-	networkName := os.Getenv(NETWORK_NAME)
-	if networkName != "" {
-		return networkName
+	if len(vipNetworkList) > 1 {
+		// Only AWS cloud supports multiple VIP networks
+		if !(IsPublicCloud() && GetCloudType() != CLOUD_AWS) {
+			return nil, fmt.Errorf("More than one network specified in VIP Network List and Cloud type is not AWS")
+		}
 	}
-	return ""
-}
+	return vipNetworkList, nil
+}*/
```

Comment:
Can you please remove the commented out code?

Reply:
Done